src/Microsoft.Health.Fhir.SqlServer/Features/Operations/JobLoggerExtension.cs

```diff
@@ -0,0 +1,107 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime;
+using Microsoft.Extensions.Logging;
+using Microsoft.Health.Fhir.SqlServer.Features.Operations;
+using Microsoft.Health.JobManagement;
+
+namespace Microsoft.Health.Fhir.SqlServer.Features.Operations
```

Comment:
Could this be in a common area where CosmosDB jobs could use it too?

Reply:
That would be applicable for the old jobs, is that the intension?

Reply:
Here is an example on the *new* job framework that occurs on CosmosDB. 

https://github.com/microsoft/fhir-server/blob/53f5153126e604573441c61a82b02d713b9c6ebe/src/Microsoft.Health.Fhir.CosmosDb/Features/Operations/Export/CosmosExportOrchestratorJob.cs#L25